tofhir-server/src/main/scala/io/tofhir/server/endpoint/SchemaDefinitionEndpoint.scala

```diff
@@ -114,10 +114,10 @@ class SchemaDefinitionEndpoint(schemaRepository: ISchemaRepository, mappingRepos
     }
   }
 
-  private def getSchemaByUrl(projectId: String, url: String): Route = {
+  private def getSchemaByUrl(url: String): Route = {
     get {
       complete {
-        service.getSchemaByUrl(projectId, url) map {
+        service.getSchemaByUrl(url) map {
```

Comment:
I guess this is because schema urls are already unique reagardless of project.

Reply:
Yes, that's right.